app/views/partials/channels-tab-url-rewriting.html

```diff
@@ -0,0 +1,109 @@
+      <label><span style="color: #a94442;">*</span> From Host</label>
+      <input ng-model="newUrlRewrite.fromHost" class="form-control" placeholder="From Host" type="text" />
+      <div class="has-error-msg"><i class="glyphicon glyphicon-warning-sign"></i>{{validationRequiredMsg}}</div>
+    </div>
+    <div class="form-group" style="float: left; margin-left: 20px;" ng-class="{ 'has-error' : ngErrorUrlRewrite.fromPort }">
+      <label><span style="color: #a94442;">*</span> From Port</label><br />
+      <input ng-model="newUrlRewrite.fromPort" class="form-control" placeholder="From Port" type="text" />
+      <div class="has-error-msg"><i class="glyphicon glyphicon-warning-sign"></i>{{ngErrorUrlRewrite.portError}}</div>
+    </div>
+  </div>
+  <div class="form-group-2columns">
+    <div class="form-group" style="float: left;" ng-class="{ 'has-error' : ngErrorUrlRewrite.toHost }">
+      <label><span style="color: #a94442;">*</span> To Host</label>
+      <input ng-model="newUrlRewrite.toHost" class="form-control" placeholder="To Host" type="text" />
+      <div class="has-error-msg"><i class="glyphicon glyphicon-warning-sign"></i>{{validationRequiredMsg}}</div>
+    </div>
+    <div class="form-group" style="float: left; margin-left: 20px;" ng-class="{ 'has-error' : ngErrorUrlRewrite.toPort }">
+      <label><span style="color: #a94442;">*</span> To Port</label><br />
+      <input ng-model="newUrlRewrite.toPort" class="form-control" placeholder="To Port" type="text" />
+      <div class="has-error-msg"><i class="glyphicon glyphicon-warning-sign"></i>{{ngErrorUrlRewrite.portError}}</div>
+    </div>
+  </div>
+  <div class="form-group-2columns">
+    <div class="form-group" style="float: left;">
+      <label>
+        Path Transform
+        <i ng-show="uiSettings.showTooltips" tooltip-html-unsafe="Transform the path by using the following substitute expression format:<br /> s/from/to/g" class="glyphicon glyphicon-question-sign"></i>
+      </label><br />
+      <input ng-model="newUrlRewrite.pathTransform" class="form-control" placeholder="s/from/to/g " type="text" />
+    </div>
+  </div>
+  
+
+  <div style="margin-top: 10px;">
+    <button type="button" class="btn btn-default" ng-click="cancelUrlRewriteAddEdit()">Cancel</button>
+    <button type="button" class="btn btn-primary" ng-click="saveUrlRewrite( oldUrlRewriteIndex )" style="margin-left: 5px;">Set URL Rewrite</button>
+
+    <!-- hasErrors Alerts -->
+    <alert style="display: inline-block; margin: 0px 0px 0px 5px; padding: 8px;" ng-repeat="alert in alerts.hasErrorsUrlRewrite" type="alert.type">
+      <i class="glyphicon glyphicon-warning-sign"></i>{{alert.msg}}
+    </alert>
+    <!-- hasErrors Alerts -->
+  </div>
+</div>
+
+
+<div ng-if="!urlRewriteAddEdit">
+
+  <div class="form-group-2columns">
+    <div class="form-group" style="float: left;">
+      <label>URL Rewriting enabled?</label><br />
```

Comment:
Perhaps add a tooltip here with the the following text: "URL rewriting allows the OpenHIM to look for URLs in a response and rewrite them so that they point to the correct location. These rules can be specified explicitly below."